web/samples_index/lib/src/samples.yaml

```diff
@@ -653,3 +653,19 @@ samples:
     web: https://flutter-angular.web.app/
     type: sample
 
+  - name: Next Gen UI demo
+    author: Flutter
+    screenshots:
+      - url: images/next_gen_ui_demo.png
+        alt: Next Gen UI demo screenshot
+    source: https://github.com/flutter/samples/tree/main/next_gen_ui_demo
+    description: A sample version of the Next Generation UI codelab for demoing at I/O Connect
+    difficulty: advanced
+    packages: 
+      - flutter_animate
+    platforms:
+      - Windows, macOS, Linux, web, Android, iOS
+    links:
+      - https://codelabs.developers.google.com/codelabs/flutter-next-gen-uis
```

Comment:
Links are a map with two entries,`text` and `href`. https://github.com/flutter/samples/blob/main/web/samples_index/lib/src/data.dart#L178

The [GitHub Action](https://github.com/flutter/samples/actions/runs/5011332922/jobs/8982046277) shows the exception:

```
ParsedYamlException: line 669, column 7 of lib/src/samples.yaml: Unsupported value for "links". type 'String' is not a subtype of type 'Map<dynamic, dynamic>' in type cast
    ╷
669 │ ┌       - https://codelabs.developers.google.com/codelabs/flutter-next-gen-uis
670 │ │     type: sample
    │ └────^
    ╵
#0      checkedYamlDecode (package:checked_yaml/checked_yaml.dart:48:5)
#1      getSamples (package:samples_index/samples_index.dart:18:15)
<asynchronous suspension>
#2      generate (file:///home/runner/work/samples/samples/web/samples_index/tool/grind.dart:46:17)
<asynchronous suspension>
#3      _RootZone.bindUnaryCallbackGuarded.<anon> (dart:async/zone.dart:1633:12)
<asynchronous suspension>
  Failed! (1) – samples_index – `flutter pub run grinder deploy`
```

Reply:
Looking at the content of the config file it looks like zero entries currently use the `links` attribute. Is this a capability that is unsupported, or should be sunset due to lack of use?

Also, debugging something that happens after commit is less than optimal.